src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/group/RabbitMessageGroupBatchSubscriber.kt

```diff
@@ -33,7 +36,22 @@ class RabbitMessageGroupBatchSubscriber(
     override fun getDeliveryCounter(): Counter = INCOMING_MSG_GROUP_BATCH_QUANTITY
     override fun getContentCounter(): Counter = INCOMING_MSG_GROUP_QUANTITY
     override fun getProcessingTimer(): Histogram = MSG_GROUP_PROCESSING_TIME
-    override fun extractCountFrom(message: MessageGroupBatch): Int = message.groupsCount
+
+    private fun getSessionAliasAndDirection(messageID: MessageID): Array<String> {
+        return arrayOf(messageID.connectionId.sessionAlias, messageID.direction.name)
+    }
```

Comment:
Maybe we need to extract this method into a utility class. We use the same logic for other routers